src/Statistic.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Secret Santa project.
+ *
+ * (c) JoliCode <coucou@jolicode.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace JoliCode\SecretSanta;
+
+use Predis\Client;
+
+class Statistic
```

Comment:
This name does not really reflect the intent.
I would name it `StatisticCollector`